bot/cogs/filtering.py

```diff
@@ -27,6 +27,7 @@
     flags=re.IGNORECASE
 )
 
+SPOILER_RE = re.compile(r"(\|\|.+?\|\|)", re.DOTALL)
```

Comment:
Okay, I got two issues with this:
1) It's a bit strict. For example, if there are only pipes before but not after the word, this will break. If the words are wrapped in single pipes instead of doubles, this will break. Perhaps it would be better to get rid of _an arbitrary amount of pipes_, or to just get rid of all the pipes in the entire word before filtering.

2) Do we really need regex for just getting rid of pipes? maybe we should solve this with just content.replace or something.

Reply:
This pattern is used for expanding a message that may contain a spoilered word, like what gdude mentioned in the issue. For actually getting rid of pipes in watchlist triggers, etc. I'm just using [`discord.utils.escape_markdown`](https://discordpy.readthedocs.io/en/latest/api.html#discord.utils.escape_markdown)